<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ViewEportSwcToFile</class>
 <widget class="QDialog" name="ViewEportSwcToFile">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>860</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ViewEportSwcToFile</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Save Path:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="SavePath"/>
         </item>
         <item>
          <widget class="QPushButton" name="SelectSavePathBtn">
           <property name="text">
            <string>Select Save Path</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QFormLayout" name="formLayout">
         <property name="labelAlignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <item row="0" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Select Swc You want to export:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QListWidget" name="SwcList"/>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <layout class="QFormLayout" name="formLayout_2">
         <item row="0" column="0">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Result</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QTableWidget" name="ResultTable"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="ExportBtn">
           <property name="text">
            <string>Export</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="CancelBtn">
           <property name="text">
            <string>Cancel</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
